CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributing
```

Comment:
I kind of messed up my fork on this one. I'm still a little new to pull requests and this was not well performed on my part.